Makefile

```diff
@@ -4,6 +4,7 @@ GOVET=$(GOCMD) vet
 BINARY_NAME=kube-ns-suspender
 VERSION=$(shell git describe --tags)
 DOCKER_REGISTRY?=ghcr.io/govirtuo
+BUILD_DATE=$(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
❓ / 🔧 The `BUILD_DATE` has not the same format than in GitHub actions

```
date +'%Y-%m-%d_%H:%M:%ST%Z'
```

https://github.com/govirtuo/kube-ns-suspender/blob/main/.github/workflows/publish_image.yaml#L41
https://github.com/govirtuo/kube-ns-suspender/blob/main/.github/workflows/test-k8s.yaml#L87
https://github.com/govirtuo/kube-ns-suspender/blob/main/.github/workflows/test-k8s.yaml#L223